### PR TITLE
[BugFix] react-native package old peer dependency issue

### PR DESCRIPTION
**Pull request by desusai7**
## Description of the change

Updated Peer Dependency of react-native package to '>=0.41.2 <=0.67.2'

## Type of change
- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#86 ]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "prettier": "^1.14.2"
   },
   "peerDependencies": {
-    "react-native": "0.41.2"
+    "react-native": ">=0.41.2 <=0.67.2"
```
Comment (pallabmaiti) [inline package.json:17]: Did you test with 0.41.2?

**Review by pallabmaiti — APPROVED** [review]

**Comment by WorldSeso7** [comment]
The 1.1.2 version lets me to install and integrate in my app, but the package is not working.
In a simple case I added the sdk and tested some events but nothing arrives on the live event tab on the control panel.
I tested with the test script and it's working as expected, so I think it's a problem of the react native SDK.

**Comment by desusai7** [comment]
Can you please  try setting the log level to verbose and share it with us, so we can further investigate it ?

**Comment by WorldSeso7** [comment]
I added the rudderClient in this way:
```
await rudderClient.setup(WRITE_KEY, {
        dataPlaneUrl: DATA_PLANE_URL,
        trackAppLifecycleEvents: true,
        recordScreenViews: true,
        logLevel: RUDDER_LOG_LEVEL.VERBOSE
      });
```

The screen method is called like this:
```
rudderClient.screen(screenClass, {
      screen_class: screenClass,
      screen_name: description,
    }).then(() => console.log('screen promise then')).catch((error) => console.log('screen promise catch', error));
```

The track method is called like this:
`
rudderClient.track(_event, data).then(() => console.log('track promise then')).catch((error) => console.log('track promise catch', error));
 `

Here's the log of a test navigation...

info Reloading app...
 BUNDLE  ./index.js 

 LOG  RudderSDK: Debug: Initializing Rudder RN SDK version: 1.1.2
 LOG  RudderSDK: Debug: setup: created config
 LOG  RudderSDK: Debug: setup: setup completed
 LOG  screen promise then
 LOG  track promise then
 LOG  screen promise then
 LOG  screen promise then
 LOG  track promise then
 LOG  screen promise then
 LOG  screen promise then
 LOG  track promise then
 LOG  screen promise then
 LOG  track promise then
 LOG  screen promise then
 LOG  screen promise then
 LOG  track promise then
 LOG  screen promise then
 LOG  track promise then
 LOG  screen promise then
 LOG  screen promise then

**Comment by WorldSeso7** [comment]
@desusai7 any news about this?
We've problems with a client for this

**Comment by desusai7** [comment]
Hey @WorldSeso7 , Can you please join slack community [here](https://www.rudderstack.com/join-rudderstack-slack-community/) and DM me. (Slack ID: Desu Sai Venkat).
Would be more than happy to help you solve this quicker.

**Comment by WorldSeso7** [comment]
The issue is solved, thank you @desusai7 for the support.

**Pull request merged**